### PR TITLE
Distinct link text

### PR DESCRIPTION
**Pull request by alwerner**

### CODE REVIEW

File: docs/consume-events.md [hunk 1]
```diff
@@ -6,24 +6,25 @@ title: Consuming Events
...
 
 ### Determine if you need an ESECC request
 
-See the [ESECC section](./administrative-requirements.md#esecc) on the Administrative Requirements page.
+See the [ESECC section on the Administrative Requirements page](./administrative-requirements.md#esecc).
 
 ### For consumers of BIP-sourced Events
 
-Read the [documentation on requesting data access and sensitivity filtering](./administrative-requirements.md#consumers-of-bip-sourced-events) in our Administrative Requirements section.
+Read the [documentation on requesting data access and sensitivity filtering on our Administrative Requirements page](./administrative-requirements.md#consumers-of-bip-sourced-events).
```
Comment (christineyost) [inline docs/consume-events.md:27]:
```suggestion
Read the [documentation about requesting data access and sensitivity filtering on our Administrative Requirements page](./administrative-requirements.md#consumers-of-bip-sourced-events).
```

File: docs/produce-events.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ Partitions in Kafka serve as the primary unit of storage within a topic, with ea
 
 #### Event retention
 
-Event retention refers to how long an event exists within Kafka and remains available for consumption. This setting would be especially important to consumers who need to be prepared to handle missed events before they expire. For additional information and discussion, see our section about Retention in the [Event Design ADR (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Engineering/ADR/ADR%20event%20design.md).
+Event retention refers to how long an event exists within Kafka and remains available for consumption. This setting would be especially important to consumers who need to be prepared to handle missed events before they expire. For additional information and discussion, see our section about Retention in our [Event Design Architectural Decision Record (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Engineering/ADR/ADR%20event%20design.md).
```
Comment (christineyost) [inline docs/produce-events.md:37]:
```suggestion
Event retention refers to how long an event exists within Kafka and remains available for consumption. This setting would be especially important to consumers who need to be prepared to handle missed events before they expire. For additional information and discussion, see the section about Retention in our [Event Design Architectural Decision Record (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Engineering/ADR/ADR%20event%20design.md).
```

File: docs/use-events.md [hunk 3]
```diff
@@ -18,23 +18,23 @@ In the Event Catalog, you’ll find the following information for every event:
...
 
 ![A table displaying all events in the Event Catalog on the CODE VA website. The headers are name, topic, lifecycle, domain, and actions. Filters allow content in the table to be filtered.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/18290306/c50c04e0-0f98-4901-81f8-0b3c9d709137)
 
 ![An event details page on the CODE VA website. The page gives an overview of one event, including the information listed above.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/18290306/c6f32b9f-475c-4426-aeeb-65bb2cf8d6a4)
 
 ## Producing and consuming events from the catalog
 
-Producers add events to the catalog and are responsible for keeping them up to date. You can read more about what to expect as a producer on the [producing events](produce-events.md) page.
+Producers add events to the catalog and are responsible for keeping them up to date. You can read more about what to expect as a producer on the [producing events page](produce-events.md).
```
Comment (christineyost) [inline docs/use-events.md:29]: Happened to notice "producing events" in similar links was capitalized, like `Producing Events page`. Should we be consistent about if these page titles are capitalized or not? Same for "consuming events" below

Comment (alwerner) [reply docs/use-events.md]: The page names have been capitalized.

**Review by christineyost — APPROVED** [review]

**Pull request merged**